### ISSUE TITLE
Train Code

### ISSUE THREAD

**Issue by ZhouCX117** [yifanw90/UMNet#1]
Hi, thanks for your excellent work! Could you please share your training code? I want to have a further understanding of the loss function and the generation process for the uncertainty map. Looking forward to your answer. Thanks again!

**Comment by yifanw90** [comment]
Thanks for your attention. We will reorganize and release the training code in the near future.

**Comment by Taoww21480** [comment]
Thank you very much for your work. I have benefited a lot from it. Therefore, could you please provide the complete code, especially for the training section and UMNet? Looking forward to your reply. Thank you very much!

**Comment by yifanw90** [comment]
>



> Thank you very much for your work. I have benefited a lot from it. Therefore, could you please provide the complete code, especially for the training section and UMNet? Looking forward to your reply. Thank you very much!

We have provided the training code. Thanks for your interest.
